configure.ac

```diff
@@ -75,6 +73,21 @@ AC_SEARCH_LIBS([cos], [m], [], [
 AC_LANG_POP([C])
 
 # ---------------------------------------------
+# enable fortran interfaces
+# ---------------------------------------------
+FORTRAN_INTERFACES=0
+AC_ARG_ENABLE([fortran-interfaces], AC_HELP_STRING([--enable-fortran-interfaces],[enable fortran support]),
+[
+#AC_PROG_FC
```

Comment:
These are commented out the way you have them, I believe. You'll need to uncomment them for the Fortran checks to be done.